src/main/webapp/charts.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Movie Information Charts</title>
+    <script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
+    <script type="text/javascript">
+        google.charts.load('current', {packages: ['corechart']});
+        google.charts.setOnLoadCallback(drawChart);
+        function drawChart(){
+          fetch("/moviechart").then((response) => response.json())
+            .then((movieJson) => {
+              var movieData = new google.visualization.DataTable();
+              //define columns for the DataTable instance
+              movieData.addColumn('string', 'Movie Title');
+              movieData.addColumn('number', 'Audience Score %');
+              movieData.addColumn('number', 'Worldwide Gross ($)');
+              movieData.addColumn('string', 'Genre');
+              movieData.addColumn('number', 'Profitability');
+
+              for (i = 0; i < movieJson.length; i++) {
+                movieRow = [];
+                movieRow.push(movieJson[i]['Film'], Number(movieJson[i]['Audience score %']),
+                              Number(movieJson[i]['Worldwide Gross']), movieJson[i]['Genre'],
+                              Number(movieJson[i]['Profitability']));
+
+                movieData.addRow(movieRow);
+              }
+              var chartOptions = {
+                title: 'Correlation between Audience Score, Worldwide Gross and Profitability of some movies',
+                hAxis: {title: 'Audience Score', viewWindow: {min: 30, max: 100}},
+                vAxis: {title: 'Worldwide Gross'},
+                bubble: {textStyle: {fontSize: 11}}
+              };
+              var movieChart = new google.visualization.BubbleChart(document.getElementById('movie_chart'));
+              movieChart.draw(movieData, chartOptions);
+            })
+        }
+    </script>
+  </head>
+  <body>
+    <h1>Charts</h1>
+    <div id="movie_chart" style="width: 1400px; height: 1000px;"></div>
```

Comment:
Would be better to add some CSS and style it using CSS classes. Also, defining the width and height will result in scrolling when viewing the site in different browser sizes (viewports, like phones, tablets, etc.)

Reply:
The chart display doesn't really scale well because there are too many data points so I would rather viewers to scroll, but maybe not the best solution

Reply:
Ah we can revisit this when working on the UI